instruqt/proxy/02-kurl-install/assignment.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: kurl-install
+id: c0g5jayumshi
+type: challenge
+title: kURL install with proxy config
+teaser: Install kURL cluster via proxy server
+notes:
+- type: text
+  contents: |
+    # Install kURL using a proxy server
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+difficulty: basic
+timelimit: 3600
+---
+
+💡 Shell
+=========
+
+Check proxy env variables present
+```
+env | grep proxy
+```
+
+Download kurl installer
+```
+curl --proxy http://${LOCAL_PROXY_IP}:3128 https://kurl.sh/proxy-$INSTRUQT_PARTICIPANT_ID -o /root/kurl-install.sh
+```
+
+Install kurl kubernetes cluster
+note the below command uses 'screen' to run the install in the background which protects instruqt session timeouts when screen locks.
+```
+screen -d -m bash -c 'time bash /root/kurl-install.sh 2>&1 | tee -a /root/kurl-install.log'
+```
+
```

Comment:
I've used `tmux` to similar ends in the airgap lab, we should look at consolidating to one approach down the road. For now, we should merge this as is.

Reply:
I guess I've used screen before in anger, it does the job for now, but happy to discover tmux later and change for consistency across the labs if tmux is the preferred choice